code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -701,17 +701,17 @@ datum/status_effect/stabilized/blue/on_remove()
 /datum/status_effect/stabilized/sepia/on_apply()
 	if(ishuman(owner))
 		var/mob/living/carbon/human/H = owner
+		owner.next_move_modifier *= 0.7
+		owner.action_speed_modifier *= 0.7
```

Comment:
Pretty sure action speed modifier is also in physiology, please use that

Reply:
no its in the mob/click 

![image](https://github.com/yogstation13/Yogstation/assets/75333826/0d9d4b8e-8c06-46fc-865a-e8d12c48986b)

Reply:
so it doesn't allow me to adjust it with .physiology

Reply:
Pretty sure its called something different
Also i said action speed, not next mov3

Reply:
![image](https://github.com/yogstation13/Yogstation/assets/75333826/eedf2845-ca1a-4843-8494-4042cc601c54)

Reply:
![image](https://github.com/yogstation13/Yogstation/assets/75333826/67aee809-ac18-42f2-acf5-f5c657af0bfd)

Reply:
nvm i found it 

![image](https://github.com/yogstation13/Yogstation/assets/75333826/13384849-13fd-4ecb-bbf2-d54bbee82d3b)